FordLabs/PeopleMover
# 213 move to openshift v4

## Issue
Connects #213

## What was done
- [x] Make new jenkinsfile in the root of our project that runs on an openshift4 version of jenkins
- [x] Fix lint warnings that showed up when running and building react app

## How to test
1. Run stage branch pipeline on new jenkins and make sure everything deploys okay
